services/app-api/utils/validation/schemaMap.ts

```diff
@@ -157,6 +157,7 @@ export const nested = (
     mixed: number(),
     string: string(),
     date: date(),
+    object: object(),
```

Comment:
Devnotes: For handling dropdowns in nested fields. It automatically detects it from the generic `object` type